### ISSUE TITLE
transforms.Normalize ?

### ISSUE THREAD

**Issue by WERush** [BoyuanJiang/context_encoder_pytorch#4]
Hi,
In this [line](https://github.com/BoyuanJiang/context_encoder_pytorch/blob/master/train.py#L79), the mean and std is (0.5, 0.5, 0.5), but the pixel of image is 0~255. Is it right? In my opinion, the mean and std should be (127.5, 127.5, 127.5). I am confused and hope you can help me.
Thanks.

**Comment by BoyuanJiang** [comment]
@WERush 
HI,because we use **transforms.ToTensor()** ,which will convert image array from [0,255] to [0,1].

http://pytorch.org/docs/master/torchvision/transforms.html#torchvision.transforms.ToTensor

**Comment by WERush** [comment]
Thanks, I get it.
